### DISCUSSION TITLE
hookah-air-purifier

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#31] [Blog comments]
# hookah-air-purifier

Обожаю кальяны, но их курение 4-5 шт в сутки в 25 кв/м помещении без вентиляции приводит к некоторым проблемам

https://blog.amd-nick.me/hookah-air-purifier

**Comment by mvrq** [comment]
Думаю взять такой же девайс для очистки воздуха к съемной квартире, есть пара вопросов)

Проблемы со следами на стенах/мебели, очиститель решил?
Удаляет ли эта штука все запахи после кальяна, или все же остается какой-то запах после курения и помещение пахнет кальяном?

**Reply by AMD-NICK** [reply]
Спустя вот эти пару месяцев или уже даже год как у меня есть этот очиститель могу добавить немного инфы.

1. Сменные фильтры стоят довольно дорого, поэтому мы стараемся выжимать из каждого максимум. Это приводит к тому, что из фильтра спустя время начинает сочиться часть дыма
2. Иногда очиститель просто "лень" включать. Так что я не могу оценить на 100% разницу "с ним" и "без".
3. Не весь дым выдыхается в фильтр. Вернее прямиком в фильтр ничего не выдыхается. Он просто стоит в комнате и фильтрует то, что в ней. Так что сама комната все равно в дыму, просто его намного меньше в итоге.

У нас, несмотря на фильтр, мебель, стекла, потолки.. Все это натягивает на себя мелассу. Можно проводить регулярную уборку, но это геморрой само собой. Мебель запах натягивает, но я не чувствую, зато чувствуют те, кто с кальянами дела не имеет. У меню нюх на них уже атрофирован )

В целом полностью заныкать покуры получится только при условии, что фильтр новый (пережил не более 50-100 покуров) и выдыхать будешь именно в него. Если будешь лениться это делать, то комната непременно будет натягивать все на себя.

Спустя столько времени мы все равно время от времени пользуемся фильтром, потому что дышать в маленькой комнате все равно намного легче с ним.

И вот еще. Я несколько раз пытался найти другое решение проблемы с кальянным дымом. Искал отдельно воздушные двигатели (типа как на скрине) + картриджный hepa фильтр (потому что именно hepa фильтр впитывает в себя мелассу)

![screenshot_2023-12-02_16 56 06@2x](https://github.com/AMD-NICK/blog.amd-nick.me/assets/9200174/9e1293dc-b079-43e5-9ee8-4a9c3abc6cec)

Но я не нашел хепа фильтры под 125 диаметр.


Сейчас еще подумываю над тем, чтобы купить рекуператор. Это система, которая втягивает воздух с улицы и вытягивает на улицу из дома. Но это нужно иметь не съемную квартиру. Хотя решение кажется мне довольно перспективным. Если не забуду и куплю, то поделюсь инфой помогает ли

**Reply by mvrq** [reply]
Спасибо за развернутый ответ, будем думать)
